app/scripts/controllers/pod.js

```diff
@@ -316,6 +317,17 @@ angular.module('openshiftConsole')
           }
         };
 
+        var focusTerminal = function() {
+          $('.terminal').focus();
```

Comment:
Good catch. It looks like we keep the terminal once it's been used, so it's possible to have multiple. I'll change it to only focus the visible terminal.